.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Docker Images & Push to Registry
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Frontend Docker image
+      working-directory: ./app
+      run: |
+        docker login docker.pkg.github.com -u ${{ secrets.USERNAME }} --password ${{ secrets.GITHUB_TOKEN }}
+        docker build . --file Dockerfile --tag frontend
+        docker tag frontend docker.pkg.github.com/googleinterns/major-tom/frontend
```

Comment:
Correct me if I am wrong but, are these URL's correct? I thought that for pushing to Google's Container Registry you had to prefix your images with the following [format](https://cloud.google.com/container-registry/docs/pushing-and-pulling):
```
gcr.io/$PROJECT_ID/$IMAGE_NAME
i.e. gcr.io/major-tom-123/frontend
```

Reply:
I think this pr is about publishing our latest image to the repo not to gcp's container registry